addons/pos_restaurant/models/pos_config.py

```diff
@@ -25,6 +26,11 @@ def _onchange_module_pos_restaurant(self):
             'is_table_management': False,
             'iface_orderline_notes': False})
 
+    @api.onchange('iface_tipproduct')
+    def _onchange_iface_tipproduct(self):
+        if not self.iface_tipproduct:
+            self.update({ 'set_tip_after_payment': False })
```

Comment:
you can just write 'self.set_tip_payment = False'

Reply:
Indeed.